gulp-tasks/copy.js

```diff
@@ -1,31 +1,52 @@
-'use strict';
-
-// Include gulp
 const { src, dest } = require('gulp');
-
-// Include Our Plugins
 const filter = require("gulp-filter");
 const rename = require('gulp-rename');
+const changed = require("gulp-changed");
+const dutil = require("./utils/doc-util");
 
-// Export our tasks.
 module.exports = {
 
+  // Copy documentation
+  copyDocs() {
+    const docs = ["README.md", "LICENSE.md", "CONTRIBUTING.md"];
+
+    dutil.logMessage("docs", "Copying documentation dist dir");
+    return src(docs).pipe(dest("dist"));
+  },
+
   // Copy vendor sass to library
   copyVendor() {
+    const vendorSass = "src/patterns/stylesheets/lib";
+
     return src('./node_modules/normalize.css/normalize.css')
+      .pipe(changed(vendorSass))
       .pipe(filter("**/normalize.css", { restore: true }))
       .pipe(rename("_normalize.scss"))
-      .pipe(dest('src/patterns/stylesheets/lib'));
+      .pipe(dest(vendorSass));
   },
 
   // Copy Sass to dist folder
   copySass() {
+    dutil.logMessage("copyDistSass", "Copying all Sass to dist dir");
+
     return src('src/patterns/**/**/*.scss')
       .pipe(dest('dist/scss'));
   },
 
+  /**
+   * Todo: remove once issue below is resolved
+   * Patternlab doesn't move CSS from `dist/` to `build/`.
+   * Alternative is run `rebuildPL` task, but that takes 11s on average
+   * https://github.com/pattern-lab/patternlab-node/issues/1310
+   */
+  copyCSSToPL() {
```

Comment:
Created `copyCSSToPL()` because of a pattern lab issue with compiles. This just copies the compiled `dist/` directory and moves it into `build`.  It gets called in `watch.js`.